keyboards/hotdox76v2/hotdox76v2.c

```diff
@@ -0,0 +1,308 @@
+// Copyright 2021 JasonRen(biu)
```

Comment:
Is it intentional to have different copyright holder here (and in the other files below)?